documentation/jetty-documentation/src/main/asciidoc/old_docs/security/authentication.adoc

```diff
@@ -511,3 +511,19 @@ Below is an example which, like the one above, sets up a server with a `HashLogi
 ----
 include::{SRCDIR}/examples/embedded/src/main/java/org/eclipse/jetty/embedded/SecuredHelloHandler.java[]
 ----
+
+==== JSR 196: Java Authentication Service Provider Interface for Containers (JASPI)
+
+Jetty can utilize portable authentication modules that implements the Jakarta Authentication specification. This requires the jetty-jaspi module.
+
+Only modules conforming to the ServerAuthModule interface in the https://www.jcp.org/en/jsr/detail?id=196[JASPI Spec] are supported. These modules must be configured before start-up.
```

Comment:
In this version of jetty, it's the old 1.0.0 jaspi spec - can we find a link direct to that version?

Reply:
I did have a look to find a direct version but didn't find one. You need to click the checkbox to agree before you can view it.